e2e/pages/enterFamilyMan/enterFamilyMan.js

```diff
@@ -0,0 +1,12 @@
+const I = actor();
+
+module.exports = {
+    
+  fields: {
+    caseId: '#familyManCaseNumber',
+  },
+
+  enterCaseID(caseId = 'mock case ID') {
+    I.fillField(this.caseIdField, caseId);
```

Comment:
does this work? youre using this.caseIdField, shouldnt it be `this.fields.caseId`?